app/app.go

```diff
@@ -297,6 +311,15 @@ func New(
 		appCodec, keys[ibchost.StoreKey], app.GetSubspace(ibchost.ModuleName), app.StakingKeeper, app.UpgradeKeeper, scopedIBCKeeper,
 	)
 
+	// register the proposal types
+	govRouter := govtypes.NewRouter()
+	govRouter.AddRoute(govtypes.RouterKey, govtypes.ProposalHandler).
+		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
+		// TODO: do we want a community pool at all? If not, does it even make sense to include that NewCommunityPoolSpendProposalHandler?
```

Comment:
Actually. Let's remove this TODO. We should rather track this in an issue if anything. We had a community pool all along btw (with no governance to govern the tokens in that pool 😬).
```suggestion
		// TODO: do we want a community pool at all? If not, does it even make sense to include that NewCommunityPoolSpendProposalHandler?
```